cfgov/v1/atomic_elements/organisms.py

```diff
@@ -560,3 +560,21 @@ class VideoPlayer(blocks.StructBlock):
     class Meta:
         icon = 'media'
         template = '_includes/organisms/video-player.html'
+
+
+class HTMLBlock(blocks.StructBlock):
+    html_url = blocks.RegexBlock(
+        label='Source URL',
+        default='',
+        required=True,
+        regex=r'^https://.+$',
```

Comment:
the regex for this can be updated once we know where the files will live and their subdomain.